browser/scripts/texture.js

```diff
@@ -36,15 +36,27 @@ Texture.prototype.load = function(src, core)
 	
 	img.onerror = function(self, src, c) { return function()
 	{
-		var dt = c.renderer.default_tex;
 
-		msg('ERROR: Failed to load texture \'' + src + '\'', 'Renderer');
-		c.asset_tracker.signal_failed();
+		var fqdn = '/* @echo FQDN */'; // Fill in FQDN (fully qualified domain name) from gulpfile for the player.
+		// gulp-preprocess replaces above with the string 'undefined' if gulpfile didn't provide the FQDN for some reason.
+		// Inside the editor, the string replace is not evaluated so we need to check against the preprocessing string as well.
+		if(fqdn === 'undefined' || fqdn === '/* @echo FQDN */') fqdn = 'vizor.io'; // defaulting to vizor.io
+
```

Comment:
I know this was there earlier, but doesn't this always make it vizor.io btw? if I set FQDN to 'foo' then fqdn === 'foo' and it gets set to vizor.io? maybe check indexOf('FQDN') instead?